Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+
+$tech-logo-size: 100px;
+$green: #BED23C;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.row {
+    display: flex;
+    flex-flow: row nowrap; 
+    height: 100%;
+}
+
+.col {
+    border: 2px solid $green;
+    border-radius: 5px;
+    margin: 1em;
+    min-height: 350px;
+    padding: 1em;
+    position: relative;
+    width: 24%;
+}
+
+.square {
+    background-color: $green;
+    border-radius: 5px;
+    min-height: 350px;
+    right: -12px;
+    left: 12px;
+    padding: 1em;
```

Comment:
use 2 space indentation in this whole file. we also don't use `em` as a unit since we prefer to keep absolute values that can represent the design properly.